OpenOversight/app/templates/auth/email/change_password.html

```diff
@@ -0,0 +1,11 @@
+<p>Dear {{ user.username }},</p>
+<p>Your password has just been changed.</p>
+<p>If you initiated this change to your password, you can ignore this email.</p>
+<p>
+  If you did not reset your password, please contact <a href="mailto:{{ help_email }}">the OpenOversight help account</a>; they will help you address this issue.
+</p>
+<p>Sincerely,</p>
+<p>The OpenOversight Team</p>
+<p>
+  <small>Note: replies to this email address are not monitored.</small>
```

Comment:
Another small rephrasing to make this potentially more generic across other instances.

```suggestion
  <small>Note: replies to this email address may not be monitored.</small>
```

Reply:
This was just piggy backing on previous emails. I'd rather not change it, because I'd need to change the other emaisl as well.

Reply:
I changed it to `Please note that we may not monitor replies to this email address.` so that we are using active voice. It has been added to all the emails.